clar2wasm/src/words/blockinfo.rs

```diff
@@ -316,6 +466,177 @@ mod tests {
                 .to_string()
                 .contains("expecting 2 arguments, got 3"));
         }
+
+        #[test]
+        fn get_tenure_info_less_than_two_args() {
+            let result = evaluate("(get-tenure-info? id-header-hash)");
+            assert!(result.is_err());
+            assert!(result
+                .unwrap_err()
+                .to_string()
+                .contains("expecting 2 arguments, got 1"));
+        }
+
+        #[test]
+        fn get_tenure_info_more_than_two_args() {
+            let result = evaluate("(get-tenure-info? id-header-hash u0 u0)");
+            assert!(result.is_err());
+            assert!(result
+                .unwrap_err()
+                .to_string()
+                .contains("expecting 2 arguments, got 3"));
+        }
+
+        #[test]
+        fn get_stacks_block_info_id_header_hash() {
+            let mut env = TestEnvironment::new(StacksEpochId::Epoch30, ClarityVersion::Clarity3);
+            env.advance_chain_tip(1);
+            let result = env
+                .evaluate("(get-stacks-block-info? id-header-hash u0)")
+                .expect("Failed to init contract.");
+            let mut expected = [0u8; 32];
+            hex::decode_to_slice(
+                "b5e076ab7609c7f8c763b5c571d07aea80b06b41452231b1437370f4964ed66e",
```

Comment:
Where does this hash come from?

Reply:
The hash here is taken from an older test for `get-block-info?`. I did this because the logic behind `get-tenure-info? id-header-hash` and `get-block-info? id-header-hash` are the same and both use the `get_index_block_header_hash` function. Based on this I assumed it should return the same value.

Reply:
Don’t say that to me, write a comment and explain this to the next person who will read your code.

Reply:
Got it, I'll put a comment with the explanation.